server/src/main/java/org/apache/calcite/server/ServerDdlExecutor.java

```diff
@@ -154,6 +156,13 @@ static Pair<CalciteSchema, String> schema(CalcitePrepare.Context context,
     return Pair.of(schema, name);
   }
 
+  static Table table(CalcitePrepare.Context context, SqlIdentifier id) {
```

Comment:
Could we give some code comments for this method?It is better to help others understand this method.

Reply:
Fixed.